iFixit/react-commerce
# Revert back to original header logo

### Overview:

![original-blue](https://user-images.githubusercontent.com/95656772/187986112-7188b683-da79-4dc6-afda-98f70c907793.jpg)

The original blue iFixit logo 🔵 is replacing the UA logo 🇺🇦 

---

### Changes Made:
1. Reverted https://github.com/iFixit/react-commerce/pull/563

---

### CR/QA Notes:

>**CR**: Review to https://github.com/iFixit/react-commerce/pull/676/commits/317e29996f9b2e1b493174646aab5b0eb7cafc9f

>**QA**:
>
>1. Navigate to [`revert-ua-logo`](https://react-commerce-prod-git-revert-ua-logo-ifixit.vercel.app/Parts)
>2. Ensure that the header logo is as shown above on /Parts and /Tools

---

cr_req 2
Connects [#444](https://github.com/iFixit/ifixit/issues/44461)
